tests/performance_estimation/DLE/test_dle.py

```diff
@@ -424,3 +444,21 @@ def test_binary_classification_result_plots_raise_no_exceptions(estimator_args,
         _ = sut.plot(**plot_args)
     except Exception as exc:
         pytest.fail(f"an unexpected exception occurred: {exc}")
+
+
+def test_oo_behavior_reg_perf_est(regression_data):  # noqa: D103
```

Comment:
Name could be a bit more descriptive, as this is a weird thing to test.

Maybe something like `test_input_dataframes_are_not_altered_by_calculator`